Cargo.toml

```diff
@@ -55,10 +63,18 @@ bytes = { version = "0.5", optional = true, features = ["serde"] }
 base64 = "^0.10"
 nom = "^5.1"
 sha2 = "^0.8.0"
+cfg-if = "1.0"
+# Crypto backends
+ring = { version = "0.16", optional = true }
+openssl-sys = { version = "0.9", optional = true}
+openssl = { version = "0.10", optional = true}
+rc_crypto = { git = "https://github.com/mozilla/application-services", rev="8a576fbe79199fa8664f64285524017f74ebcc5f", optional = true, feature="gecko"}
```

Comment:
Out of curiosity, wouldn't this prevent a new release of `authenticator` from being made to crates.io because of the `git` dependencies?

Reply:
As it stands right now: Yes. But the rc_crypto-crate is also under the control of Mozilla. I briefly talked to the team maintaining that repo, when I added this dependency. They didn't want to publish it back then, but if the demand is there, they might be swayed.

Reply:
My current employer makes use of this library to power our security key integrations, so it would be great to support more of the proper FIDO2 APIs from a published crate release instead of a Git branch. This is of course assuming its not unreasonable to ask of the maintaining teams since it does incur a long-term maintenance burden.